dropwizard-core/src/main/java/io/dropwizard/setup/Environment.java

```diff
@@ -119,6 +119,8 @@ public Environment(String name,
         }
```

Comment:
This source file is a frequently changed file with significant technical debt.

                                         Leave the campground cleaner than you found it! Improvement estimate: starting at 5 mins. 
<!-- lookout footnote separator -->
_Comment made by the analyzer techdebt._